docs/modules/record-encryption/con-lost-kek.adoc

```diff
@@ -0,0 +1,94 @@
+// file included in the following:
+//
+// assemblies/assembly-operations-record-encryption-filter.adoc
+
+[id='con-lost-kek-{context}']
+= Dealing with a lost KEK
+
+This section discusses what to do if KEK that is still in use to encrypt topics is KEK is lost.
```

Comment:
should we define 'lost', with an example, like 'deleted from the KMS'. It's the first time the term has showed up. To clarify it's about the external KMS.